src/components/select/select.test.jsx

```diff
@@ -166,6 +166,13 @@ describe('src/components/select/select', () => {
       render(<Select {...requiredProps} errorText="not valid">{baseListOptionElements}</Select>);
       expect(screen.getByLabelText('Test label')).not.toBeValid();
     });
+
+    it('does not show errorText on first render until blurred', () => {
+      render(<Select {...requiredProps} required >{baseListOptionElements}</Select>);
```

Comment:
```suggestion
      render(<Select {...requiredProps} required>{baseListOptionElements}</Select>);
```

What is this, dangling space town?